Only in xbattbar-0.2/: dist
Only in xbattbar-0.2/: shell.nix
diff -ur /tmp/xbattbar-0.2/src/XBattBar/Backend.hs xbattbar-0.2/src/XBattBar/Backend.hs
--- a/src/XBattBar/Backend.hs	2012-10-08 00:53:55.000000000 +0200
+++ b/src/XBattBar/Backend.hs	2018-12-30 14:01:46.342271089 +0100
@@ -15,8 +15,8 @@
 getChargeLinux :: IO Double
 getChargeLinux = do
     let path = "/sys/bus/acpi/drivers/battery/PNP0C0A:00/power_supply/BAT0/"
-    fullS <- readFile $ path++"energy_full"
-    nowS <- readFile $ path++"energy_now"
+    fullS <- readFile $ path++"charge_full"
+    nowS <- readFile $ path++"charge_now"
     let f = read fullS
     let n = read nowS
     return (n / f)
